CHANGELOG.md

```diff
@@ -1,4 +1,47 @@
 ## 0.6.0 (Unreleased)
+
+**Implemented enhancements:**
+
+- mongodbatlas\_database\_user can not be imported when they contain dashes "-" in the name [\#179](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/179)
```

Comment:
Move this (line 5, issue #179 ) to closed issues

Reply:
okay